jetty-util/src/main/java/org/eclipse/jetty/util/BufferUtil.java

```diff
@@ -166,6 +167,23 @@ public static ByteBuffer copy(ByteBuffer buffer)
         return clone;
     }
 
+    /**
+     * Reset the buffer's endianness to {@link ByteOrder#BIG_ENDIAN}
+     * and clear the buffer to be empty in flush mode.
+     * The position and limit are set to 0;
```

Comment:
Why semicolon at end of sentence? Something was forgot or typo?

Reply:
This is an exact copy of `clear()`'s javadoc, with the extra _Reset the buffer's endianness..._ bits added on top. I'll fix both.